Ginger/GingerCoreNET/UserCofig/UserProfile.cs

```diff
@@ -432,14 +432,18 @@ public void SaveUserProfile()
         {
             try
             {
+                Console.WriteLine("Before Agent mapping");
                 SaveRecentAppAgentsMapping();
+                Console.WriteLine("After Agent mapping");
+
             }
             catch (Exception ex)
             {
                 Reporter.ToLog(eLogLevel.ERROR, "Error occurred while saving Recent App-Agents Mapping for User Profile save", ex);
             }
-
+            Console.WriteLine("Before Save file");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)